File: app/src/main/java/com/unsplash/client/network/Api/ApiPhotos.java
package com.unsplash.client.network.Api;

import com.unsplash.client.data.model.PhotoModel;
import com.unsplash.client.network.response.ApiResponse;

import java.util.List;

import retrofit2.http.GET;
import rx.Observable;

public interface ApiPhotos {

    @GET("search/photos")
    Observable<ApiResponse<List<PhotoModel>>> getPhotos(String query);
}

File: app/src/main/java/com/unsplash/client/network/Api/ApiPhotos.kt
package com.unsplash.client.network.Api

import com.unsplash.client.data.model.PhotoModel
import com.unsplash.client.network.response.ApiResponse

import retrofit2.http.GET
import retrofit2.http.Query
import rx.Observable

interface ApiPhotos {

    @GET("search/photos")
    fun getPhotos(@Query("client_id") clientId: String, @Query("page") page: Int, @Query("query") query: String): Observable<ApiResponse<List<PhotoModel>>>
}
